### PR TITLE
OpenXmlElement.CreateXmlReader: Prevent XmlConvertingReaderFactory from using a disposed TextReader

### PR DESCRIPTION
**Pull request by sorensenmatias**
OpenXmlElement.CreateXmlReader: Prevent XmlConvertingReaderFactory from using a disposed TextReader.

### CODE REVIEW

File: src/DocumentFormat.OpenXml/OpenXmlElement.cs [hunk 1]
```diff
@@ -1765,21 +1765,20 @@ private protected XmlReader CreateXmlReader()
 
         private protected XmlReader CreateXmlReader(string outerXml)
         {
-            using (TextReader stringReader = new StringReader(outerXml))
+            var stringReader = new StringReader(outerXml);
```
Comment (twsouthwick) [inline src/DocumentFormat.OpenXml/OpenXmlElement.cs:1768]: Can you add a comment here why we don't want to have a using?

**Review by twsouthwick — CHANGES_REQUESTED** [review]
Can you include a regression test for this!

Thanks for the fix!

**Comment by sorensenmatias** [comment]
@twsouthwick I added a test that shows the problem I encountered, and confirms the fix.

**Comment by sorensenmatias** [comment]
Is there anything else I can help with to get this merged?

**Review by twsouthwick — DISMISSED** [review]
LGTM!

**Comment by twsouthwick** [comment]
@sorensenmatias I've been pretty busy and so the delay is on me here. Once the tests pass, this can be merged in. Thanks for the contribution!

**Review by twsouthwick — DISMISSED** [review]

**Comment by sorensenmatias** [comment]
Thanks @twsouthwick. I am not sure I completely follow though, is there a problem with the tests currently?

**Comment by twsouthwick** [comment]
Yeah - we have warnings as error on, and there's a style warning that's getting flagged

**Comment by twsouthwick** [comment]
I've pushed a fix for it (needed to add a copyright)

**Comment by sorensenmatias** [comment]
Thanks. I fixed one more warning regarding UTF8 BOM.

**Review by twsouthwick — APPROVED** [review]

**Pull request merged**